app/src/modules/collections/routes/item.vue

```diff
@@ -338,7 +340,8 @@ export default defineComponent({
 		});
 
 		useShortcut('meta+s', saveAndStay, form);
-		useShortcut('meta+shift+s', saveAndAddNew, form);
+		useShortcut('meta+shift+s', saveAsCopyAndNavigate, form);
+		useShortcut('alt+n', createNew, form);
 
```

Comment:
@benhaynes 
Here I have separated the Save and Create New into two actions.
If you want to save you should press CMD + S.
If you want to create a new item you should press Option + N. Unfortunately, chromium based browsers does not allow to use some keybindings [source](https://stackoverflow.com/a/7296303)

The same behaviour happens on `...` menu, Save and Create New is replaced by Create New.

Reply:
Nice! Does the `ALT+N` also _save_ the current item before opening a new page? This would be the difference between these two:

* Save and Create New
* Create New

Also, is the `...` always visible now? Ideally we just disable the individual items in the context menu.

Reply:
It only takes the user to the new item page. The behaviour is similar when you are editing some item and then you quit that view. In both scenarios, if you have not saved changes, a popup will ask if you want to save those changes.
But I agree that using shortcut should not show this popup. When using shortcuts we are trying to save time, so it makes totally sense.

About the `...`, now is always visible, except for the singletons, since no actions on that menu will available. Should  it be visible regardless is singleton or not?

Reply:
Ahh, gotcha... I think we'd want to keep the current flow, which is:

1. Save the Current Item
2. Navigate to New Item Form

Coolio?

> Singleton — Should it be visible regardless is singleton or not?

Hmm, maybe we keep it there and disable the options that aren't valid? That way the user can click to see the `META-S` keyboard shortcut, and we have a menu for any other options in the future... such as "Save as Future Draft Revision", which would apply to singletons.